notifications_utils/logging.py

```diff
@@ -20,12 +20,16 @@ def init_app(app, statsd_client=None):
     app.config.setdefault('NOTIFY_LOG_LEVEL', 'INFO')
     app.config.setdefault('NOTIFY_APP_NAME', 'none')
     app.config.setdefault('NOTIFY_LOG_PATH', './log/application.log')
+    app.config.setdefault('NOTIFY_RUNTIME_PLATFORM', 'local')
```

Comment:
Optional

I'm interested in whether we should even have a default value. Could/should the default just be `None`? 

I guess it feels a bit weird that when this will be run shortly in production in the PaaS that the runtime platform will be set as `local`. Will we go and set it to be `paas` for all the apps (which feels unnecessary)?

Also `local` to me says a developer machine. Is that what you meant local to mean or do you mean local to mean running locally ie not in a docker container?

Reply:
Yeah, I wasn't particularly happy with any of the options tbh.

I think your suggestion makes sense, since we will only care about this option on ECS, we may as well set it to `None` by default, to avoid having to go and set it to `paas` for all the apps.